kf_lib_data_ingest/target_api_plugins/kids_first_dataservice.py

```diff
@@ -874,7 +874,15 @@ def coerce_types(host, entity_class, body):
                 # Don't include it so that the server applies default behavior.
                 continue
         else:
-            ret[k] = json_type_casts.get(properties[k]["type"], lambda x: x)(v)
+            v = json_type_casts.get(properties[k]["type"], lambda x: x)(v)
+            if properties[k]["type"] == "integer":
+                try:
+                    positive_bits = int(properties[k]["format"][-2:]) - 1
+                    if v > 2 ** positive_bits:
+                        v = None
```

Comment:
Should we log some sort of warning when a value becomes null?

Reply:
added